packages/ui/layout.tsx

```diff
@@ -1,9 +1,10 @@
-import React, { FC } from 'react'
+import type { FC, ReactNode } from 'react'
 import Head from 'next/head'
 import Nav, { NavProps } from './nav'
 import { Vercel } from './icons'
 
 export interface LayoutProps extends NavProps {
+  children?: ReactNode
```

Comment:
@GuiBibeau What's `PropsWithChildren`? btw this was a fix because React type for ^18 no longer include `children` in the `FC` type, I did not change this it for any other reason.